docs/reference/mapping/fields/id-field.asciidoc

```diff
@@ -3,10 +3,14 @@
 
 Each document has an `_id` that uniquely identifies it, which is indexed
 so that documents can be looked up either with the <<docs-get,GET API>> or the
-<<query-dsl-ids-query,`ids` query>>.
+<<query-dsl-ids-query,`ids` query>>. The `_id` can either be assigned at
+indexing time, or a unique `_id` can be generated by {es}. This field is not
+configurable.
```

Comment:
I don't know if `configurable` is clear here.
```suggestion
<<query-dsl-ids-query,`ids` query>>. You can specify a unique `_id` in the
indexing request or let {es} generate one. However, you can't change an existing
document's `_id`.
```

Reply:
I think that Wylie means that you con't configure the mapping of the `_id` field as opposed to the value of the `_id` field.